### ISSUE TITLE
Annotate throws exception

### ISSUE THREAD

**Issue by gregoriorobles** [se2p/LitterBox#3]
## Overview

When running LitterBox with the annotate argument, I obtain an exception.

### Further details

The exact output of the command is below:

```
$ java -jar Litterbox-1.4-SNAPSHOT.jar --check --projectid 470599232 --path subidr/ --annotate results.json
Issue dead_code was found 1 time(s)
Issue long_script was found 4 time(s)
Issue same_variable_different_sprite was found 4 time(s)
Issue unused_variables was found 2 time(s)
Issue variable_initialization_race was found 1 time(s)
Exception in thread "main" java.lang.ClassCastException: class de.uni_passau.fim.se2.litterbox.ast.model.metadata.block.NoBlockMetadata cannot be cast to class de.uni_passau.fim.se2.litterbox.ast.model.metadata.block.NonDataBlockMetadata (de.uni_passau.fim.se2.litterbox.ast.model.metadata.block.NoBlockMetadata and de.uni_passau.fim.se2.litterbox.ast.model.metadata.block.NonDataBlockMetadata are in unnamed module of loader 'app')
	at de.uni_passau.fim.se2.litterbox.report.CommentGenerator.generateReport(CommentGenerator.java:45)
	at de.uni_passau.fim.se2.litterbox.analytics.BugAnalyzer.createAnnotatedFile(BugAnalyzer.java:108)
	at de.uni_passau.fim.se2.litterbox.analytics.BugAnalyzer.check(BugAnalyzer.java:70)
	at de.uni_passau.fim.se2.litterbox.analytics.Analyzer.analyzeSingle(Analyzer.java:121)
	at de.uni_passau.fim.se2.litterbox.Main.runAnalysis(Main.java:196)
	at de.uni_passau.fim.se2.litterbox.Main.checkPrograms(Main.java:158)
	at de.uni_passau.fim.se2.litterbox.Main.parseCommandLine(Main.java:215)
	at de.uni_passau.fim.se2.litterbox.Main.main(Main.java:239)

```

The Java version I am using:

```
$ java --version
openjdk 11.0.9.1 2020-11-04
OpenJDK Runtime Environment (build 11.0.9.1+1-Ubuntu-0ubuntu1.20.04)
OpenJDK 64-Bit Server VM (build 11.0.9.1+1-Ubuntu-0ubuntu1.20.04, mixed mode, sharing)
```

**Comment by gofraser** [comment]
Thanks for reporting this!

@Hallomann93 added a fix for this in aa530d9380859f9e852a28f057e8f479351e647b. This revealed a followup problem with that particular project, which he also fixed in 7581b170a9f3141a3a96362af0ad8ee3540da685. 

Will merge into master asap, for now this is also in https://github.com/se2p/LitterBox/tree/spanish_translation

**Comment by gregoriorobles** [comment]
Works now! Thanks a lot, @gofraser and @Hallomann93
